src/js/lotto/createLotto.js

```diff
@@ -0,0 +1,12 @@
+import { LOTTO_MAX_NUMBER, LOTTO_LENGTH } from '../constants/lotto.js';
+
+const BALLS = new Array(LOTTO_MAX_NUMBER).fill(null).map((value, index) => index + 1);
+
+export default function createLotto() {
+  const balls = [...BALLS];
+  const lotto = new Array(LOTTO_LENGTH).fill(null).map(() => {
+    const ballIndex = Math.floor(Math.random() * (balls.length - 1));
+    return balls.splice(ballIndex, 1)[0];
+  });
+  return lotto;
```

Comment:
![image](https://user-images.githubusercontent.com/18749057/179386442-9d64051f-48e5-42b0-ae4b-df6ed950f6c1.png)

```suggestion
  return balls.sort(() => 0.5 - Math.random()).slice(0, 6);
```

이전 리뷰에서 제가 코드를 잘못 안내해드린 것 같네요!
이런식으로 표현할 수 있답니다 ㅎㅎ